src/com/lftechnology/java/training/sanish/palindrome/Main.java

```diff
@@ -0,0 +1,19 @@
+package com.lftechnology.java.training.sanish.palindrome;
+
+import java.util.logging.Logger;
+
+/**
+ * Class Main
+ * 
+ * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+ */
+public class Main {
+	public static void main(String args[]){
+		final Logger LOGGER = Logger.getLogger(Main.class.getName());
+		try {
+			LOGGER.info("Largest palindrome made from the product of two 3-digit numbers: "+ Palindrome.getLargestPalindrome(3));
```

Comment:
better to use parameterized logger.
http://stackoverflow.com/questions/17716381/slf4j-parameterized-logging-using-varargs-method